app/controllers/clients_controller.rb

```diff
@@ -123,15 +125,31 @@ def create
   end
 
   def update
-    if @client.update(safe_params)
+    @client = Client.where(symbol: params[:id]).first
+    exists = @client.present?
+
+    if exists
```

Comment:
Not sure what you are trying to do here. Create a new client with the update method?

Reply:
it has been removed now